build/BundledDotnetTools.props

```diff
@@ -0,0 +1,6 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <BundledDotnetTools Include="dotnet-watch" Version="$(MicrosoftAspNetCoreAllPackageVersion)" />
```

Comment:
ASP.NET need to maintain this list

Reply:
I'm okay just starting with dotnet-watch and dotnet-dev-certs.

Do we have a plan for our other cli tools? We also produce: dotnet-user-secrets, dotnet-sql-cache, dotnet-aspnet-codegeneration, dotnet-ef

Reply:
If we decide to add them as "native verbs" to the CLI, it should be a matter of just adding them here, as long as they are available as global tools packages in a feed accessible by the CLI builds.